askama_parser/src/tests.rs

```diff
@@ -25,33 +25,47 @@ fn test_invalid_block() {
 
 #[test]
 fn test_parse_filter() {
-    use Expr::*;
     let syntax = Syntax::default();
     assert_eq!(
         Ast::from_str("{{ strvar|e }}", &syntax).unwrap().nodes,
-        vec![Node::Expr(Ws(None, None), Filter("e", vec![Var("strvar")]),)],
+        vec![Node::Expr(
+            Ws(None, None),
+            Expr::Filter(Filter {
```

Comment:
Looks like these changes belong in the previous commit.

Reply:
Arf indeed! `git reflog` to the rescue!

Reply:
Rescue was a success :laughing: .